pkg/ee/default-policy-controller/controller.go

```diff
@@ -220,53 +220,140 @@ func policyBindingReconcilerFactory(template kubermaticv1.PolicyTemplate) reconc
 }
 
 // isClusterTargeted checks if the PolicyTemplate targets the given cluster.
-func isClusterTargeted(cluster *kubermaticv1.Cluster, template *kubermaticv1.PolicyTemplate) bool {
-	projectID := cluster.Labels[kubermaticv1.ProjectIDLabelKey]
+func isClusterTargeted(ctx context.Context, client ctrlruntimeclient.Client, cluster *kubermaticv1.Cluster, template *kubermaticv1.PolicyTemplate) bool {
```

Comment:
nit: if make it a pointer receiver, we can use r.client internally and save an argument spot. same for the others who take client as arguments as well..

Reply:
I made the decision to have it like this because it is used in two places, and one of those places is inside enqueueClusters, which is not a method of reconciler. I updated it now to have all of them as reconciler methods, with the reasoning being that all these funcs are using something from the reconciler, which is the client.